server/utils/taskListUtils.test.ts

```diff
@@ -15,6 +17,8 @@ describe('taskListUtils', () => {
   describe('taskLink', () => {
     describe('with an application', () => {
       const application = applicationFactory.build({ id: 'some-uuid' })
+      ;(isFullPerson as jest.MockedFunction<typeof isFullPerson>).mockReturnValue(true)
```

Comment:
I think it's best we put lines 20 and 21 in a `beforeEach` and make sure we do `jest.resetAllMocks()` before.